Sources/idt/idt.cc

```diff
@@ -253,7 +253,9 @@ class visitor : public clang::RecursiveASTVisitor<visitor> {
     // access level.
     bool should_export_record = false;
     for (const auto *MD : RD->methods())
-      if ((should_export_record = (MD->isVirtual() && !MD->hasBody())))
+      if ((should_export_record =
+               (MD->isVirtual() && !MD->isPureVirtual() && !MD->hasBody() &&
+                !MD->isDefaulted() && !MD->isDeleted())))
```

Comment:
```suggestion
               !(MD->isPureVirtual() || MD->isDefaulted() || MD->isDeleted()) &&
               (MD->isVirtual() && !MD->hasBody());
```

I think that this matches the commit message a bit more closely - not a pure virtual, defaulted, or deleted virtual method that is out of line.